app/__init__.py

```diff
@@ -1,16 +1,18 @@
 from flask import Flask, jsonify, request
 from firebase_admin import credentials, firestore, initialize_app
 
+# Initializing Firestore database
+# Can import firestore_db to utilize database
 cred = credentials.Certificate("app/key/serviceAccountKey.json")
 default_app = initialize_app(cred)
 firestore_db = firestore.client()
 
 
-def create_app():
-    app = Flask(__name__)
+# def create_app():
+#     app = Flask(__name__)
 
-    @app.route("/success")
-    def health():
-        return jsonify("success")
+#     @app.route("/success")
+#     def health():
+#         return jsonify("success")
 
-    return app
+#     return app

```

Comment:
Generally if we don't need something later you can just remove it, no need to keep it in comments.

Reply:
I will remove this. I forgot to delete after testing the new implementation.